pkg/didcomm/envelope/api.go

```diff
@@ -6,12 +6,16 @@ SPDX-License-Identifier: Apache-2.0
 
 package envelope
```

Comment:
Is it correct to observe that this package is only needed by the framework (supplying the injectable API)?

Reply:
In talking with @Baha-sk we've decided to go the other way - Packager does not need to be injectable because pack/unpack format configuration is done through `WithDefaultPacker()` and `WithPackers()`. Where do you think Packager should move? Without Packager injection we only need the interface for mocking.

Reply:
Packager will hold the list of Packers including a default one, it will also decide which one to use based on the envelope type. Users don't need to inject it as it's a utility interface that's required by the framework.

Users can inject Packers and a DefaultPacker to support their own implementation.